SimG4CMS/ShowerLibraryProducer/src/HcalForwardAnalysis.cc

```diff
@@ -129,9 +129,10 @@ void HcalForwardAnalysis::setPhotons(const EndOfEvent* evt) {
   std::vector<HFShowerPhoton> LongFiberPhotons;
   LongFiberPhotons.clear();
   ShortFiberPhotons.clear();
+  int thehc_entries = theHC->entries();
   if (idHC >= 0 && theHC != nullptr) {
-    std::cout << "FiberhitSize " << theHC->entries() << std::endl;
-    for (j = 0; j < theHC->entries(); j++) {
+    std::cout << "FiberhitSize " << thehc_entries << std::endl;
```

Comment:
cout is not ideal, but this code is likely not part of official workflows meant to run in multithreaded mode